examples/build.gradle

```diff
@@ -0,0 +1,30 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.2.3'
+    }
+}
+
+ext {
+    defaultCompileSdkVersion = 25
+    defaultMinSdkVersion = 9
```

Comment:
First I use 16, but I guessed, I will found someone who isn't happy with this. So I start with 9 to go to 16 finally :-)